components/SidebarNav/react.stories.js

```diff
@@ -1,149 +1,183 @@
+      voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    </p>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+      labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+      voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    </p>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+      labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+      voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    </p>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+      labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+      voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+    </p>
+  </div>
+)
+
+const menuItems = [
+  {
+    label: 'Exterior Doors',
+    icon: <span dangerouslySetInnerHTML={{ __html: mEntry }} />,
+    active: true,
+    href: '/',
+    items: [
       {
-        label: 'External Link',
-        icon: <span dangerouslySetInnerHTML={{ __html: mExport }} />,
-        href: '#',
+        label: 'Menu Item 1',
+        href: '/menu/1',
+        onClick,
       },
       {
-        label: 'Contact',
-        icon: <span dangerouslySetInnerHTML={{ __html: mContact }} />,
-        href: '#',
+        label: 'Menu Item 2',
+        href: '/menu/2',
+        onClick,
       },
-    ].map(addKey)
+    ].map(addKey),
+  },
+  {
+    label: 'External Link',
+    icon: <span dangerouslySetInnerHTML={{ __html: mExport }} />,
+    href: '#',
+  },
+  {
+    label: 'Contact',
+    icon: <span dangerouslySetInnerHTML={{ __html: mContact }} />,
+    href: '#',
+  },
+].map(addKey)
 
-    const bottomMenuItems = [
-      {
-        label: 'Log Out',
-        icon: <span dangerouslySetInnerHTML={{ __html: mSignout }} />,
-      },
-    ].map(addKey)
+const bottomMenuItems = [
+  {
+    label: 'Log Out',
+    icon: <span dangerouslySetInnerHTML={{ __html: mSignout }} />,
+  },
+].map(addKey)
+
+const Logo = (
+  <a href="/" tabIndex="0">
```

Comment:
Curious why we need to set tabIndex here?

Reply:
The logo prop accepts a component. It isn't an automatic link, so it needs to be provided, if it is a link.